### ISSUE TITLE
When I save settings in servers a lot of config section are also modified

### ISSUE THREAD

**Issue by noplay** [GNS3/gns3-gui#462]
```
2015-06-23 16:37:05 INFO local_config.py:230 Section Servers has changed. Saving configuration
2015-06-23 16:37:05 INFO local_config.py:131 Configuration save to /Users/noplay/.config/gns3.net/gns3_gui.conf
2015-06-23 16:37:05 INFO local_config.py:230 Section Dynamips has changed. Saving configuration
2015-06-23 16:37:05 INFO local_config.py:131 Configuration save to /Users/noplay/.config/gns3.net/gns3_gui.conf
2015-06-23 16:37:05 INFO local_config.py:230 Section IOU has changed. Saving configuration
2015-06-23 16:37:05 INFO local_config.py:131 Configuration save to /Users/noplay/.config/gns3.net/gns3_gui.conf
2015-06-23 16:37:05 INFO local_config.py:230 Section Qemu has changed. Saving configuration
2015-06-23 16:37:05 INFO local_config.py:131 Configuration save to /Users/noplay/.config/gns3.net/gns3_gui.conf
2015-06-23 16:37:05 INFO local_config.py:230 Section VirtualBox has changed. Saving configuration
2015-06-23 16:37:05 INFO local_config.py:131 Configuration save to /Users/noplay/.config/gns3.net/gns3_gui.conf
2015-06-23 16:37:05 INFO local_config.py:230 Section VMware has changed. Saving configuration
2015-06-23 16:37:05 INFO local_config.py:131 Configuration save to /Users/noplay/.config/gns3.net/gns3_gui.conf

**Comment by noplay** [comment]
I understand why. 

Take for example VMware:

The settings are like this:
```json
  "VMware": {
    "host_type": null,
    "use_local_server": true,
    "vmnet_end_range": 255,
    "vmnet_start_range": 2,
    "vmrun_path": "/Applications/VMware Fusion.app/Contents/Library/vmrun",
    "vms": []
  },
```

When we load the General VMware settings we keep only the keys available here:
```python
VMWARE_SETTINGS = {
    "vmrun_path": "",
    "host_type": "ws",
    "vmnet_start_range": 2,
    "vmnet_end_range": DEFAULT_VMNET_END_RANGE,
    "use_local_server": True,
}
```
This mean the vms key is dropped. At this point the section VMware of the settings contain:
```json
  "VMware": {
    "host_type": null,
    "use_local_server": true,
    "vmnet_end_range": 255,
    "vmnet_start_range": 2,
    "vmrun_path": "/Applications/VMware Fusion.app/Contents/Library/vmrun"
  },
```

A few line later we will recreate the vms  key and at this point the system will detect the change and save again the config (it's why you got reload loop and you dropped the save section in the VM code I think). 

Perhaps it will be simpler to never drop the unknown keys from settings. The other benefits is to avoid losing configuration when you start a dev version with additional settings and after that you use the current version.

**Comment by noplay** [comment]
Fixed by jeremy in 12975b1ecf1404563df2d0733b14633e8060a519
